content/2022-02-03-be-a-guest-blogger.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Be a Guest Blogger!"
```

Comment:
I would remove all references to "guest blogger" because spammers use that phrase a lot, so it will probably hurt your SEO. I'd phrase it as "Share about your Brigade experience" or some such, here.

Reply:
Wow. I literally had no idea about this. Thanks you for the heads up! Consider it fixed!

Reply:
Fixed in ed4554183f14ca2a968da573b77b5bdada72e949